.buildkite/core.rayci.yml

```diff
@@ -184,6 +184,16 @@ steps:
       - "3.10"
       - "3.11"
 
+  # cpp tests
+  - label: ":ray: core: cpp worker tests"
+    tags: core_cpp
+    instance_type: small
+    commands:
+      - ci/ci.sh build
+      - ci/ci.sh test_cpp
```

Comment:
do we have a plan to stop using this `ci/ci.sh` script in the future?

Reply:
Yess, there are only a few straggler remained